requirements/dev.txt

```diff
@@ -20,3 +20,6 @@ livereload==2.3.2
 
 # Avoid eating cpu with live reloading
 watchdog==0.8.3
+
+# QA Helper
```

Comment:
No idea. This didn't actually work, though, and I ended up having to move the import inside the function. I don't have any good explanation for it, but these two lines should be removable without affecting anything.